pkg/dashboard/resource/function.go

```diff
@@ -450,26 +490,49 @@ func (fr *functionResource) getFunctionReplicas(request *http.Request) (
 		return nil, errors.Wrap(err, "Failed to get function")
 	}
 
+	includeOffline := fr.GetURLParamBoolOrDefault(request, "includeOffline", false)
+
 	permissionOptions := opa.PermissionOptions{
 		MemberIds:           opa.GetUserAndGroupIdsFromAuthSession(fr.getCtxSession(ctx)),
 		OverrideHeaderValue: request.Header.Get(opa.OverrideHeader),
 		RaiseForbidden:      true,
 	}
 
-	replicaNames, err := fr.getPlatform().GetFunctionReplicaNames(ctx, function, permissionOptions)
+	activeReplicaNames, err := fr.getPlatform().GetFunctionActiveReplicaNames(ctx, function, permissionOptions)
+
 	if err != nil {
 		return nil, errors.Wrap(err, "Failed to get function replicas")
 	}
-	return &restful.CustomRouteFuncResponse{
-		Resources: map[string]restful.Attributes{
-			"replicas": map[string]interface{}{
-				"names": replicaNames,
-			},
-		},
-		Single:     true,
+
+	response := &restful.CustomRouteFuncResponse{
+		Resources:  map[string]restful.Attributes{},
+		Single:     false,
 		Headers:    map[string]string{"Content-Type": "application/json"},
 		StatusCode: http.StatusOK,
-	}, nil
+	}
+	response.Resources["replicas"] = map[string]interface{}{
+		"names": activeReplicaNames,
+	}
+	if includeOffline {
+		timeFilter := fr.getURLParamTimeFilterOrDefault("timeFilter", request, nil)
+
+		// get all replicas
+		allReplicaNames, err := fr.getPlatform().GetFunctionAllReplicaNames(ctx, function, permissionOptions, timeFilter)
+		if err != nil {
+			return nil, errors.Wrap(err, "Failed to get function replicas")
+		}
+
+		// filter out online replicas
+		offlineReplicas := fr.getOfflineReplicas(allReplicaNames, activeReplicaNames)
+		if len(offlineReplicas) > 0 {
+			response.Resources["offlineReplicas"] = map[string]interface{}{
+				"names": offlineReplicas,
+			}
+			response.Single = false
```

Comment:
not needed because it is already set to false on response initialization.

Reply:
@TomerShor chaged to `true` in initialisation